src/sidenav/components/SidenavMenu.vue

```diff
@@ -32,6 +32,47 @@
         </QBadge>
       </QItemSection>
     </QItem>
+    <QExpansionItem
+      switch-toggle-side
+      :label="$t('BUTTON.SHOW_MORE')"
+      dense
+      dense-toggle
+      expand-separator
+    >
```

Comment:
The "Show more" is not aligned with the other icons/labels which looks odd to me:

![misaligned](https://user-images.githubusercontent.com/31616/75761210-f947b780-5d38-11ea-9fd2-59e90266ad08.png)

Reply:
I think we also need to include a badge on the Show more, if it's closed and one of the items inside has a badge.

Reply:
I found a half solution for the "Show More". I could align the toggle but not really the text. But I'm too lazy to be bothered by it now. hehe

Badge included! Not the best solution if we add other items with badges in the future, but works for now

Reply:
Ok! I had a dig into the alignment issue of the "Show more", it's because in the rest of the list we are not using the "standard" layout for "list with icons" but our own combination of quasar components.

We use `side` mode for the `<QItemSection>` that contains the menu icon, but the standard approach uses the `avatar` mode. If we used `avatar` mode then all the other menu entry texts would move further to the right and appear inline:

![showmore-aligned](https://user-images.githubusercontent.com/31616/80101122-4265fc00-8571-11ea-8026-0432b19f8c99.png)

However, at some point we/somebody decided that doesn't look great with our more condensed style. So, I wouldn't suggest we switch to `avatar` spacing... but rather find a way to switch the "Read more" to use _our_ way.

... however, `QExpansionItem` doesn't give an option to control the display of the expansion arrow when using `switch-toggle-side` - it always uses `avatar` spacing.

What we _can_ do though, is:
- not use `switch-toggle-side` mode
- specify a custom #header slot (we do this already...)
- define the header bit the same way as we do for the other menu items
- include an icon

Using a horizontal ellipsis icon it would look like this:

![showmore-closed](https://user-images.githubusercontent.com/31616/80101743-1d25bd80-8572-11ea-8621-cc0c1bc8c2e6.png)

... and when open:

![showmore-open](https://user-images.githubusercontent.com/31616/80101759-231b9e80-8572-11ea-8dce-790a2d958778.png)

I quite like this approach as I can't see a way to align the arrow when using `switch-toggle-side` without using some custom override CSS, which we try and avoid.

Whaddya think?

Reply:
(this comment isn't really on the correct file, it should be on `SidenavGroupUI`)

Reply:
Ah that was the case! I think the solution is good. I was even wondering if instead of using the horizontal ellipsis we could use a plus icon (that turns into a minus when open, if it's worth the trouble).